linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_pam_enabled/ansible/shared.yml

```diff
@@ -4,29 +4,40 @@
 # complexity = low
```

Comment:
Given https://github.com/ComplianceAsCode/content/commit/e7d99c03246ebc5eacba43156987e3548a6ea73e

Ensure that `multi_platform_ubuntu` is added

Reply:
Added on rebase